scripts/lib/api/Pkg.ts

```diff
@@ -12,6 +12,16 @@
 
 import { join } from "path/posix";
 
+import { findLegacyReleaseNotes } from "./releaseNotes";
+import { removePrefix, removeSuffix } from "../stringUtils";
+import { getRoot } from "../fs";
+
+export const VALID_PACKAGE_NAMES = [
+  "qiskit",
+  "qiskit-ibm-runtime",
+  "qiskit-ibm-provider",
```

Comment:
Hardcoding the 3 values is less general than we had before. But the set is small and stable enough that I think it's best to keep it simple.